zeighami-etal-2019.ipynb

```diff
@@ -0,0 +1,954 @@
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def to_1_decimal_str(f):\n",
+    "    return str(round(f, 1))\n",
+    "\n",
+    "\n",
+    "df_age = load_ppmi_csv(utils, FILENAME_AGE)\n",
+    "df_demographics = load_ppmi_csv(utils, FILENAME_DEMOGRAPHICS)\n",
+    "\n",
+    "col_male = \"is_male\"\n",
+    "col_cohort = \"cohort\"\n",
+    "\n",
+    "dfs_summary = []\n",
+    "df_assessments: pd.DataFrame\n",
+    "for cohort_name, df_assessments in cohort_assessments_map.items():\n",
+    "\n",
+    "    subjects = df_assessments[COL_PAT_ID].drop_duplicates()\n",
+    "    subjects_pd = df_status.loc[\n",
+    "        (df_status[COL_PAT_ID].isin(subjects)) & (df_status[COL_STATUS] == STATUS_PD),\n",
+    "        COL_PAT_ID,\n",
+    "    ]\n",
+    "\n",
+    "    # the paper only report measures for the PD patients\n",
+    "    df_assessments = df_assessments.loc[df_assessments[COL_PAT_ID].isin(subjects_pd)]\n",
+    "\n",
+    "    # general demographics (baseline session only)\n",
+    "    df_summary = df_assessments.merge(df_age, on=[COL_PAT_ID, COL_VISIT_TYPE])\n",
+    "    df_demographics[col_male] = (df_demographics[COL_SEX] == SEX_MALE).apply(\n",
+    "        lambda v: 100 if v else 0\n",
+    "    )\n",
+    "    df_summary = df_summary.merge(df_demographics, on=COL_PAT_ID)\n",
+    "    df_summary = df_summary[[COL_PAT_ID, COL_AGE, col_male]]\n",
+    "    df_summary[COL_FOLLOWUP] = False\n",
+    "\n",
+    "    # combine with clinical/cognitive scores\n",
+    "    df_summary = df_summary.merge(\n",
+    "        df_assessments, on=[COL_PAT_ID, COL_FOLLOWUP], how=\"outer\"\n",
+    "    )\n",
+    "    df_summary = df_summary.drop(columns=COL_PAT_ID)\n",
+    "\n",
+    "    # append\n",
+    "    df_summary[col_cohort] = cohort_name\n",
+    "    dfs_summary.append(df_summary)\n",
+    "\n",
+    "df_summary = pd.concat(dfs_summary)\n",
+    "df_summary_means = (\n",
+    "    df_summary.groupby([col_cohort, COL_FOLLOWUP]).mean().applymap(to_1_decimal_str)\n",
+    ")\n",
+    "df_summary_stds = (\n",
+    "    df_summary.groupby([col_cohort, COL_FOLLOWUP]).std().applymap(to_1_decimal_str)\n",
+    ")\n",
+    "df_summary_stds = \" (\" + df_summary_stds + \")\"\n",
+    "df_summary_stds.loc[:, col_male] = \"\"\n",
+    "df_summary_combined = (df_summary_means + df_summary_stds).T\n",
+    "df_summary_combined = df_summary_combined.applymap(lambda x: \"-\" if \"nan\" in x else x)\n",
+    "df_summary_combined = df_summary_combined.rename(\n",
+    "    index={\n",
+    "        COL_AGE: \"Age\",\n",
+    "        col_male: \"Male (%)\",\n",
+    "        COL_UPDRS2: \"UPDRS Part II\",\n",
+    "        COL_UPDRS3: \"UPDRS Part III\",\n",
+    "        COL_ADL: \"S&E ADL\",\n",
+    "        COL_PIGD: \"PIGD\",\n",
+    "        COL_UPDRS1: \"UPDRS Part I\",\n",
+    "        COL_MOCA: \"MoCA\",\n",
+    "        COL_GCO: \"Global composite outcome\",\n",
+    "    }\n",
+    ")\n",
```

Comment:
Maybe add a sentence or two to comment on how this table compares to the one in the original paper.